docs/lib/analytics/fragments/configure.md

```diff
@@ -0,0 +1,20 @@
+Existing Amazon Pinpoint resources can be used with AWS Amplify through the `awsconfiguration.json` file.
```

Comment:
is it awsconfiguration.json or amplifyconfiguration.json?

Reply:
We should probably also provide the corresponding content here: https://docs.amplify.aws/cli/usage/files#amplifyconfigurationjson-file

Reply:
What do you mean by "corresponding content?" And yes, that's my timebox showing ;) Thanks!